### PR TITLE
Goals

### PR DESCRIPTION
**Pull request by nojaf**
We were thinking of outlining our goals a bit.
This is a first attempt, feel free to nitpick and suggest other ideas you had.

### CODE REVIEW

File: src/pages/donate.astro [hunk 1]
```diff
@@ -18,21 +21,105 @@ import PageLayout from "../layouts/PageLayout.astro";
...
+      Now we are getting somewhere. We could organise a small conference with
+      this kind of money.
+    </p>
+    <ul>
+      <li>In person conference/workshop/hackaton.</li>
+      <li>Long term paid open-source F# work.</li>
+    </ul>
+  </div>
+  <div class="tier" style="background: var(--cyan-900); color: var(--white)">
+    <h3>Castle tier (50K - ...)</h3>
```
Comment (edgarfgp) [inline src/pages/donate.astro:73]: Possibility of having a foundation. So we are tax-efficient ?

**Review by TheAngryByrd — APPROVED** [review]

**Review by edgarfgp — APPROVED** [review]

File: src/pages/donate.astro [hunk 2]
```diff
@@ -18,21 +21,109 @@ import PageLayout from "../layouts/PageLayout.astro";
...
+    <p>
+      This is the lowest tier and would allow us to cover our current costs.
+    </p>
+    <ul>
+      <li>
+        Services for our online sessions (for example our Zoom subscription).
+      </li>
+      <li>Travel costs when meeting the F# community in the real world.</li>
+      <li>Buying and sending out swag like coffee mugs & t-shirts.</li>
+      <li>Buy a open-source maintainer a lunch after a great release.</li>
```
Comment (dawedawe) [inline src/pages/donate.astro:43]:
```suggestion
      <li>Buy an open-source maintainer a lunch after a great release.</li>
```

File: src/pages/donate.astro [hunk 3]
```diff
@@ -18,21 +21,109 @@ import PageLayout from "../layouts/PageLayout.astro";
...
+    <h3>Villa tier (5K - 15K)</h3>
+    <p>
+      This tier allows us to start paying F# contributors to work on a small
+      scoped topic.
+    </p>
+    <ul>
+      <li>Write more get started documentation for popular projects.</li>
+      <li>Contribute a new feature to the F# compiler or to tooling.</li>
+      <li>
+        Do some invisible maintance work that just cannot be done after hours.
```
Comment (dawedawe) [inline src/pages/donate.astro:57]:
```suggestion
        Do some invisible maintenance work that just cannot be done after hours.
```

File: src/pages/donate.astro [hunk 4]
```diff
@@ -18,21 +21,109 @@ import PageLayout from "../layouts/PageLayout.astro";
...
+    </ul>
+  </div>
+  <div class="tier" style="background: var(--cyan-700); color: var(--cyan-50)">
+    <h3>Mansion tier (15K - 50K)</h3>
+    <p>
+      Now we are getting somewhere. We could organise a small conference with
+      this kind of money.
+    </p>
+    <ul>
+      <li>In person conference/workshop/hackaton.</li>
```
Comment (dawedawe) [inline src/pages/donate.astro:68]:
```suggestion
      <li>In person conference/workshop/hackathon.</li>
```

**Review by dawedawe — APPROVED** [review]

**Pull request merged**